gradle/dependency-management/kotlin-versions.properties

```diff
@@ -1,2 +1,2 @@
 # Generated - Update by running `./gradlew updateKotlinVersions`
-latests=1.6.10,1.6.21,1.7.0,1.7.22,1.8.0,1.8.22,1.9.0,1.9.25,2.0.0,2.0.20
+latests=1.6.10,1.6.21,1.7.0,1.7.22,1.8.0,1.8.22,1.9.0,1.9.25,2.0.0,2.0.21-RC,2.1.0-Beta1
```

Comment:
❓When do we remove versions from this list again?
![image](https://github.com/user-attachments/assets/4d9a2cb8-764e-4022-81d4-70d49db1ea71)